import.go

```diff
@@ -46,13 +51,24 @@ type ListImportsResponse struct {
 	Response
 }
 
+// CreateImportURL creates a new import URL.
+// Note: Do not use this.
+// It is present for internal usage only.
+// This function can, and will, break and/or be removed at any point in time.
+func (c *Client) CreateImportURL(ctx context.Context, filename string) (*CreateImportURLResponse, error) {
+	var resp CreateImportURLResponse
+	err := c.makeRequest(ctx, http.MethodPost, "imports_url", nil, map[string]string{"filename": filename}, &resp)
+
+	return &resp, err
+}
+
 // CreateImport creates a new import task.
 // Note: Do not use this.
 // It is present for internal usage only.
 // This function can, and will, break and/or be removed at any point in time.
-func (c *Client) CreateImport(ctx context.Context, filename string) (*CreateImportResponse, error) {
+func (c *Client) CreateImport(ctx context.Context, path string) (*CreateImportResponse, error) {
```

Comment:
`path` standard package name, might be better to have a different name

Reply:
surprised that the linter doesn't catch this 🤔

Reply:
because it's not imported in that file at the moment but it could be later on

Reply:
ok, let me think, 'cause filepath is also taken already 😅

Reply:
`fp` 😉

Reply:
also you need to change in `CreateImportURL` #consistency